src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.SslProtocols.cs

```diff
@@ -121,7 +121,7 @@ public async Task GetAsync_AllowedSSLVersion_Succeeds(SslProtocols acceptedProto
 #if !NETFRAMEWORK
                     handler.SslProtocols = SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12 | SslProtocols.Tls13;
 #else
-                    handler.SslProtocols = SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
+                    handler.SslProtocols = SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12 | (SslProtocols)12288;
```

Comment:
```suggestion
                    handler.SslProtocols = SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12 | (SslProtocols)0x3000;
```

Reply:
I think the reason for `12288` is that's what we use in the definition:
https://github.com/dotnet/runtime/blob/9f6da559949d4f591ff87b1cef543278997d5576/src/libraries/System.Net.Primitives/ref/System.Net.Primitives.cs#L524-L524

Reply:
Ah, that is rather ugly. I wonder why? Perhaps it was auto-generated at some point?
I would still consider that hex values are more developer-friendly for reading ...

BTW: Definition is technically hex-defined:
https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/src/libraries/System.Net.Primitives/src/System/Net/SecureProtocols/SslEnumTypes.cs#L18
https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/src/libraries/Common/src/Interop/Windows/SChannel/Interop.SchProtocols.cs#L31-L33